### PR TITLE
remove hardcoded timeout

### PR DESCRIPTION
**Pull request by mattonem**

### CODE REVIEW

**Comment by mattonem** [comment]
From my observation, by removing the `this.timeout()` line, the arg `--test-timeout` starts working again. But the browserstack-sdk adds 30 seconds to the timeout regardless. So that's something we need to be aware of.

**Comment by mattonem** [comment]
Well actually the SDK add 30 minutes timeout... That's is insane. 
So we need to force timeout with a reasonable value.

**Comment by mattonem** [comment]
Okay `--test-timeout` should work now

File: browserstack-mocha-export.mjs [hunk 1]
```diff
@@ -41,10 +41,33 @@ function beforeEach() {
 
 codeExport.opts.hooks.beforeEach = new exporter.hook(beforeEach())
 
+function declareVariables() {
+    const params = {
+        startingSyntax: {
+          commands: [
+            {
+                level: 0,
+                statement: `this.timeout(process.env.testTimeout)`,
```
Comment (hrishi1096) [inline browserstack-mocha-export.mjs:50]: 🙌 Awesome

**Pull request merged**